Lesson78

gloomy-onion opened the pull request.

gloomy-onion committed:
Добавила роутер, прелоудер

gloomy-onion committed:
сделала пустую корзину

gloomy-onion committed:
сделала корзину которая будет не пустая, но пока пиццы туда не добавлены

gloomy-onion committed:
сделала фильтр по категориям пицц
сделала сортировку

Alterqq commented on src/MainContent/MainContent.jsx:
```diff
@@ -1,32 +1,54 @@
+    const category = categoryId > 0 ? `category=${categoryId}` : '';
+    const sortBy = sortType.sortProperty.replace('-', '');
+    const order = sortType.sortProperty.includes('-') ? 'asc' : 'desc';
+    fetch(`https://64c7f223a1fe0128fbd57bb6.mockapi.io/pizza?
```
Вынести урл в константы

Alterqq commented on src/MainContent/MainContent.jsx:
```diff
@@ -1,32 +1,54 @@
+        <h2 className={styles.content__title}>Все пиццы</h2>
+        <div className={styles.content__items}>
+          {isLoading
+            ? [...new Array(6)].map((_, index) => <Skeleton key={index}/>)
```
Скелетон - отдельный компонент

Alterqq commented on src/Sort/Sort.jsx:
```diff
@@ -1,28 +1,34 @@
   const [open, setOpen] = useState(false);
-  const list = ['популярности', 'цене', 'алфавиту'];
-  const [selected, setSelected] = useState(0);
+  const list = [
```
вынести в константы

Alterqq commented on src/Sort/Sort.jsx:
```diff
@@ -1,28 +1,34 @@
-          {list.map((name, i) => (
-            <li key={name} onClick={() => onClickList(i)}
-                className={selected === i ? styles.active : ''}>{name}</li>
+          {list.map((obj, i) => (
```
нейминг переменых

gloomy-onion committed:
исправила ошибки
переместила корзину
сделала константы на урл и лист
переименовала переменные
вынесла скелетон

Alterqq commented on src/common/constants.js:
```diff
@@ -0,0 +1,9 @@
+export const API_URL = 'https://64c7f223a1fe0128fbd57bb6.mockapi.io/'
+
+export const sortList = [
```
Это все же не в common должно лежать
а в src/Sort/constants.ts

Alterqq commented on src/Cart/Cart.jsx:
```diff
@@ -30,7 +30,7 @@ const Cart = () => {
               </div>
               <div className={styles.cart__bottom_buttons}>
-                <a href="/" className={cn(styles.button, styles.goBackBtn)}>
+                <a href="/src/pages" className={cn(styles.button, styles.goBackBtn)}>
```
А эт че такое?)

gloomy-onion replied:
Ой а это говно какое то вылезла, исправлю

Alterqq replied:
Там, наверное, имелось в виду "Link", а не "a"?

gloomy-onion committed:
исправила ошибки
